src/components/PrivateRoute.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```

Comment:
Any reason to use this instead of just: import React from 'react'?

Reply:
Just copied it from somewhere... changed now